WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderSubsActivity.java

```diff
@@ -82,6 +85,7 @@ protected void onCreate(Bundle savedInstanceState) {
         restoreState(savedInstanceState);
 
         mViewPager = (WPViewPager) findViewById(R.id.viewpager);
+        mViewPager.setOffscreenPageLimit(NUM_TABS - 1);
```

Comment:
Ah, okay I see where you're coming from with avoiding `NUM_TABS`. But looking at the ViewPager source code, it's better to call `setOffscreenPageLimit` before the adapter is assigned if you know the number of pages. Setting the offscreen page limit calls the ViewPager's [populate](http://grepcode.com/file/repository.grepcode.com/java/ext/com.google.android/android/4.0.1_r1/android/support/v4/view/ViewPager.java#607) method, which does a lot of extra work when the adapter has been assigned,